invenio_cli/cli/assets.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-Cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module to ease the creation and management of applications."""
+
+import click
+from click_default_group import DefaultGroup
+
+from ..commands import AssetsCommands
+from .utils import pass_cli_config, run_steps
+
+
+@click.group()
+def assets():
+    """Statics and assets management commands."""
+
+
+@assets.command()
+@click.option('--force', '-f', default=False, is_flag=True,
+              help='Force the full recreation the assets and statics.')
+@click.option(
+    '--production/--development', '-p/-d', default=True, is_flag=True,
+    help='Production mode copies files. Development mode symlinks files.'
+)
+@pass_cli_config
+def update(cli_config, force, production):
+    """Updates the current application static/assets files."""
+    commands = AssetsCommands(cli_config)
+    commands.update_statics_and_assets(
+        force=force,
+        flask_env='production' if production else 'development'
+    )
+
+
+@assets.command()
+@click.argument('path', type=click.Path(exists=True))
+@pass_cli_config
+def install(cli_config, path):
+    """Install and link a React module."""
```

Comment:
Comment: Not sure, but perhaps this should mention it's fro development?

Reply:
Done, specified for all assets and watch commands.